java/src/jmri/jmrit/withrottle/UserInterface.java

```diff
@@ -416,18 +416,16 @@ public void serviceQueued(ZeroConfServiceEvent se) {
     public void servicePublished(ZeroConfServiceEvent se) {
         try {
             try {
-                InetAddress addr = se.getAddress();
-                // most addresses are Inet6Address objects, 
-                if (addr instanceof Inet4Address && !addr.isLoopbackAddress()) {
+                InetAddress addr = se.getDNS().getInetAddress();
+                //output last good ipV4 address to the window to support manual entry
+                if (addr instanceof Inet4Address) {
```

Comment:
I just couldn't see any way for a loopback address to make it this far, (they are skipped on the publishing side), so I removed the check.